namespaces/live.cloud-platform.service.justice.gov.uk/laa-crime-hardship-test/resources/pingdom.tf

```diff
@@ -0,0 +1,15 @@
+resource "pingdom_check" "laa-crime-hardship-dev" {
```

Comment:
This should be laa-crime-hardship-test not dev.

Reply:
Good spot, updated to correct env =)